### PR TITLE
Handle the refresh with default value.

### PR DESCRIPTION
**Pull request by Abdelrahmanrezk**
Work on the comments related to change the data type of the password and to use default values for the refresh password.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -8,12 +8,15 @@
 
 from flask import Flask, jsonify, request
 from flask_cors import CORS
-
+from flask import current_app as app
```
Comment (AliOsm) [inline app.py:11]: `app` already exists in this file, no need to import it.

File: app.py [hunk 2]
```diff
@@ -8,12 +8,15 @@
...
 from utils.common_utils import dict_filter
 from utils.dataset_utils import refresh_masader_and_tags
+
+
```
Comment (AliOsm) [inline app.py:15]: Any reason for these new lines?

File: app.py [hunk 3]
```diff
@@ -8,12 +8,15 @@
...
 from utils.gh_utils import create_issue
 
 
+
```
Comment (AliOsm) [inline app.py:19]: Any reason for this new line?

File: app.py [hunk 4]
```diff
@@ -77,6 +80,8 @@ def get_tags():
     return jsonify(dict_filter(tags, features))
 
 
+
+
```
Comment (AliOsm) [inline app.py:84]: Any reason for these new lines?

File: app.py [hunk 5]
```diff
@@ -107,5 +115,4 @@ def refresh():
 
 
 with app.app_context():
-    refresh()
-    refresh()
+    refresh(app.config['REFRESH_PASSWORD'])
```
Comment (AliOsm) [inline app.py:118]: We should call `refresh` two times here to solve the empty Redis cache issue in the first startup of the application.

**Review by AliOsm — COMMENTED** [review]
Try to push your changes to the same PR instead of creating new one.

**Comment by zaidalyafeai** [comment]
This has been idle for a while, @Abdelrahmanrezk, are you still working on this?

File: app.py (hunk 5 above)
Comment (Abdelrahmanrezk) [reply app.py:118]: Yes @zaidalyafeai , should I handle the comments that @AliOsm mentioned above, Inshallah will end from that by tomorrow.

File: app.py (hunk 3 above)
Comment (Abdelrahmanrezk) [reply app.py:19]: I have not import that line ```from utils.gh_utils import create_issue``` !!

**Review by AliOsm — APPROVED** [review]

**Pull request merged**